site/en/docs/web-platform/compute-pressure/index.md

```diff
@@ -1,264 +1,221 @@
-entirely idle. A CPU utilization close to `1.0` is very likely to
-generate excessive heat. Web applications can help avoiding bad
-user experiences by reducing their compute demands when the CPU utilization is
-high.
-
-## CPU clock speed
-
-Modern CPU cores support a set of clock speeds. The device's firmware or
-operating system can set the core clock speed in order to trade off the
-available CPU computational resources with power consumption.
-
-From a user experience standpoint, the following are the most interesting clock
-speeds:
-
-- The minimum clock speed, which results in the lowest power consumption.
-- The base clock speed, which results in a moderate level of power consumption
-  and heat generation. This tends to be the speed advertised in marketing
-  materials.
-- The maximum clock speed, sometimes marketed as "turbo boost." Running at this
-  speed causes unsustainable amounts of heating. It can only be used for short
-  periods of time, to satisfy bursts in demand for computing power.
-
-When a device's CPU utilization gets high, the device may increase clock speeds
-across its CPU cores, in an attempt to meet the demand for computing power. As
-the speeds exceed the base clock speed, the elevated power consumption increases
-the CPU's temperature. At some point, the device enters a
-[thermal throttling regime](https://en.wikipedia.org/wiki/Dynamic_frequency_scaling),
-where the CPU clock speed is reduced in order to bring the temperature down.
-
-As with CPU utilization, values from `0.0` (minimal) to `1.0` (boosted) are used
-to normalize clock speeds. A value of `0.5` represents the base clock speed for
-a given CPU.
-
-## Using the API
-
-The following code creates a `ComputePressureObserver`, configured with specific
-triggering thresholds, along with an associated callback function that may be
-invoked when a threshold is crossed. The callback is intentionally rate-limited,
-with the current implementation maxing out at one invocation per second for a
-page in the foreground, and once per ten seconds for background pages.
-
-Inside the callback function, your web application can respond to the changing
-utilization and clock speed levels by enabling or disabling CPU-intensive
-functionality.
-
-```javascript
-function callback(update) {
-  if (update.cpuSpeed > 0.5) {
-    // The CPU is running at faster than base speed.
-  } else {
-    // The CPU is running at normal or reduced speed.
-  }
-
-  if (update.cpuUtilization >= 0.9) {
-    // CPU utilization is over 90%.
-  } else if (update.cpuUtilization >= 0.75) {
-    // CPU utilization is over 75%.
-  } else if (update.cpuUtilization >= 0.5) {
-    // CPU utilization is over 50%.
-  } else {
-    // CPU utilization is under 50%.
-  }
-}
+Create the pressure observer by calling its constructor with a callback function to be run whenever
+there is a pressure update:
+
+```js
+let observer = new PressureObserver(callback, { sampleRate: 0.5 });
```

Comment:
Instead of `callback`, maybe `(records) => { /* ... */ }` to make it clear.

Reply:
ok